static/i18n/ar/app.json

```diff
@@ -2908,11 +2908,10 @@
     "placeholder": "أدخل عنوان {{currencyName}}"
   },
   "Terms": {
-    "title": "شروط الاستخدام",
-    "read": "قراءة شروط الاستخدام",
-    "switchLabelTerms": "<0>لقد قرأت ووافقت على </0><1>شروط الاستخدام</1>",
-    "switchLabelPrivacyPolicy": "<0>لقد قرأت ووافقت على</0> <1>سياسة الخصوصية</1>",
-    "switchLabelLoss": "أفهم أنني إذا فقدت عبارة الاسترداد هذه، فلن أتمكن من الوصول إلى أصولي المشفرة في حالة فقداني القدرة على الوصول إلى جهاز نانو.",
+    "title": "Legal",
```

Comment:
maybe remove these changes to other languages than english :sweat_smile: 
You're invalidating translations here.
As a rule of thumbs you should never edit translation files unless its the `en` one to edit or add onto it

Reply:
Got it!

Reply:
Done -> [fee0525542d7fcc174bf95556bb9d7475605c4c0](https://github.com/LedgerHQ/ledger-live-desktop/pull/3967/commits/fee0525542d7fcc174bf95556bb9d7475605c4c0)